screens/product-list/product-list.html

```diff
@@ -0,0 +1,381 @@
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Diamond
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-4" class="center-content">
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Rings
+                            </label>
+                        </li>
+                    </ul>
+                </section>
+                <section class="rating m-xs-all text-grey-500">
+                    <div>
+                        <h4>Ratings</h4>
+                    </div>
+                    <ul class="list">
+                        <li class="list__input m-xs-all">
+                            <label for="id-1" class="center-content">
+                                <input id="id-1" type="radio" class="m-xs-r" name="I1" checked>
+                                4 star & above
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-2" class="center-content">
+                                <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                3 star & above
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-2" class="center-content">
+                                <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                2 star & above
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-2" class="center-content">
+                                <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                1 star & above
+                            </label>
+                        </li>
+                    </ul>
+                </section>
+
+                <section class="sortBy m-xs-all text-grey-500">
+                    <div>
+                        <h4>Sort by</h4>
+                    </div>
+                    <ul class="list">
+                        <li class="list__input m-xs-all">
+                            <label for="id-1" class="center-content">
+                                <input id="id-1" type="radio" class="m-xs-r" name="I1" checked>
+                                Price - Low to High
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-2" class="center-content">
+                                <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                Price - High to Low
```

Comment:
- Here you have given id as 1 & 2  but you have already given these same id's to ratings
- So suggestion here is to give some different id's here as id's should be unique.